lib/taskana-data/pom.xml

```diff
@@ -0,0 +1,55 @@
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<artifactId>taskana-data</artifactId>
+
+	<name>${project.groupId}:${project.artifactId}</name>
+	<description>The helper module to import data for testing.</description>
+
+	<parent>
+		<groupId>pro.taskana</groupId>
+		<artifactId>taskana-lib-parent</artifactId>
+		<version>1.1.6-SNAPSHOT</version>
+		<relativePath>../pom.xml</relativePath>
+	</parent>
+
+	<dependencies>
+
+		<dependency>
+			<groupId>org.mybatis</groupId>
+			<artifactId>mybatis</artifactId>
+			<version>${version.mybatis}</version>
+		</dependency>
+
+		<dependency>
+			<groupId>org.slf4j</groupId>
+			<artifactId>slf4j-api</artifactId>
+			<version>${version.slf4j}</version>
+		</dependency>
```

Comment:
please add slf4j-simple to prevent [this](https://stackoverflow.com/questions/7421612/slf4j-failed-to-load-class-org-slf4j-impl-staticloggerbinder) error.

Reply:
thx will fix this.

Reply:
done

Reply:
It is worth noting that embedded components such as libraries or frameworks should never declare a dependency on any SLF4J binding.  This is because when a library declares a compile-time dependency on an SLF4J binding, it imposes that binding on the end-user. Obviously, this negates SLF4J's basic purpose. Consequently, they should only depend on the slf4j-api library.

https://www.baeldung.com/slf4j-classpath-multiple-bindings

Reply:
this creates conflicts with other modules (rest-spring)

Reply:
added simple for the test scope

Reply:
I am a little confused now. Maybe talk later / tomorrow briefly about it?